webapp/channels/src/packages/mattermost-redux/src/actions/channels.test.ts

```diff
@@ -2099,14 +2099,13 @@ describe('Actions.Channels', () => {
         });
 
         nock(Client4.getBaseRoute()).get(
-            '/users/some-user-id/channel_members?page=0&per_page=200').
-            reply(200, [...Array(200).keys()].map((index) => ({channel_id: `channel-${index}`, user_id: 'some-user-id'})));
-        nock(Client4.getBaseRoute()).get(
-            '/users/some-user-id/channel_members?page=1&per_page=200').
-            reply(200, [...Array(200).keys()].map((index) => ({channel_id: `channel-${index + 200}`, user_id: 'some-user-id'})));
-        nock(Client4.getBaseRoute()).get(
-            '/users/some-user-id/channel_members?page=2&per_page=200').
-            reply(200, [...Array(100).keys()].map((index) => ({channel_id: `channel-${index + 400}`, user_id: 'some-user-id'})));
+            '/users/some-user-id/channel_members?page=-1&per_page=60').
```

Comment:
nit: why is per_page 60 here, shouldn't the test be calling with only the page param?

Reply:
True! Will change.

Reply:
Oh I realized what this is. This mocks the actual request that gets sent out. The `getAllChannelMembers` is implemented like this: `getAllChannelsMembers = (userId: string, page = 0, perPage = PER_PAGE_DEFAULT) => {`. So if you don't set `perPage`, it gets set to 60. That's why the test has to use `per_page=60`.